### PR TITLE
Jcampos/update dashboard help

### PR DESCRIPTION
**Pull request by jicampos**
The help icon on the dashboard opens to GitHub unless a window is in full-screen. If in full-screen, the help button displays tooltip.

### CODE REVIEW

File: WebGUI/js/Desktop.js [hunk 1]
```diff
@@ -2473,6 +2473,27 @@ Desktop.handleWindowHelp = function (mouseEvent)
 	return false;
 } //end handleWindowHelp()
 
+//==============================================================================
+Desktop.handleDashboardHelp = function (mouseEvent) 
+{
+	// Check if window exists 
+	if(Desktop.desktop.getForeWindow() == 0)
+	{
+		window.open("https://github.com/art-daq", "_blank")
```
Comment (rrivera747) [inline WebGUI/js/Desktop.js]: Please make the default ? link this: https://github.com/art-daq/otsdaq#readme

**Review by rrivera747 — CHANGES_REQUESTED** [review]

File: WebGUI/js/Desktop.js (hunk 1 above)
Comment (jicampos) [reply WebGUI/js/Desktop.js]: Updated.
https://github.com/art-daq/otsdaq_utilities/pull/38/commits/7462e3442777cf9c883e08c18d6df236f8abbd50

File: WebGUI/js/Desktop.js [hunk 2]
```diff
@@ -2473,6 +2473,27 @@ Desktop.handleWindowHelp = function (mouseEvent)
...
+//==============================================================================
+Desktop.handleDashboardHelp = function (mouseEvent) 
+{
+	// Check if window exists 
+	if(Desktop.desktop.getForeWindow() == 0)
+	{
+		window.open("https://github.com/art-daq/otsdaq#readme", "_blank")
+	}
+
+	if(Desktop.desktop.getForeWindow().isMaximized())
```
Comment (rrivera747) [inline WebGUI/js/Desktop.js]: I think you need "else if" here

Comment (rrivera747) [reply WebGUI/js/Desktop.js]: otherwise getForeWindow() will have a JS runtime error, right?

**Review by rrivera747 — CHANGES_REQUESTED** [review]

**Review by rrivera747 — APPROVED** [review]
Looks like an important improvement.

**Pull request merged**